torrent.go

```diff
@@ -2350,15 +2350,28 @@ func (t *Torrent) callbacks() *Callbacks {
 	return &t.cl.config.Callbacks
 }
 
-func (t *Torrent) AddWebSeeds(urls []string) {
+type AddWebSeedsOptions struct {
```

Comment:
I don't see any example use of this in this PR, but maybe using the function-opt pattern is more forward compatible here: `type WebseedOpt func(*webseedOpts)`, and provide `PathEscaper(pathComps []string) string` as the only instance of it for now. Then AddWebSeeds takes `...WebseedOpt` or `...AddWebseedOpt`.

Reply:
Done https://github.com/afjoseph/torrent/commit/a79c3bd5d30d99fb33ab7c5334c7d017884d4403

Reply:
I've threaded this through completely in a fix up.